integration/command.go

```diff
@@ -162,21 +162,36 @@ kubectl rollout status -n security-profiles-operator ds spod --timeout=180s || \
 }
 
 // CleanupInspektorGadget cleans up inspector gadget in Kubernetes
-var CleanupInspektorGadget *Command = &Command{
+var CleanupInspektorGadget = &Command{
 	Name:    "CleanupInspektorGadget",
 	Cmd:     "$KUBECTL_GADGET undeploy",
 	Cleanup: true,
 }
 
 // CleanupSPO cleans up security profile operator in Kubernetes
-var CleanupSPO *Command = &Command{
-	Name: "RemoveSecurityProfilesOperator",
-	Cmd: `
-	kubectl delete seccompprofile --all --all-namespaces
-	kubectl delete -f https://raw.githubusercontent.com/kubernetes-sigs/security-profiles-operator/v0.4.3/deploy/operator.yaml --ignore-not-found
-	kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/v1.8.0/cert-manager.yaml --ignore-not-found
-	`,
-	Cleanup: true,
+var CleanupSPO = []*Command{
+	{
+		Name: "RemoveSecurityProfilesOperator",
+		Cmd: `
+		kubectl delete seccompprofile --all --all-namespaces
+		kubectl delete -f https://raw.githubusercontent.com/kubernetes-sigs/security-profiles-operator/v0.4.3/deploy/operator.yaml --ignore-not-found
+		kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/v1.8.0/cert-manager.yaml --ignore-not-found
+		`,
+		Cleanup: true,
+	},
+	{
+		Name: "PatchSecurityProfilesOperatorProfiles",
```

Comment:
> I have only one concern, is it guaranteed that `PatchSecurityProfilesOperatorProfiles` will always be executed after `RemoveSecurityProfilesOperator`?

By after you mean the case in which we patch the profiles even before termination has started? How about we only do this patching once all the `SPOD` pods are gone so we are sure we really removing the dangling finalizers?

Reply:
> How about we only do this patching once all the SPOD pods are gone so we are sure we really removing the dangling finalizers?
>
That's is what I mean 🙂. I would like to be sure that we patch the SPs after we run `RemoveSecurityProfilesOperator` (`SPOD` pods are gone). Otherwise, I remember you told me that the `SPOD` will add again the finalizers. I am concern about this because the slice doesn't guarantee the order.

However, maybe your second idea is more robust ... What if it runs in the background and keep patching the SPs until all the SPs are gone? In this way, it doesn't matter whether it is executed before or after the `SPOD` is gone.

Reply:
> However, maybe your second idea is more robust ... What if it runs in the background and keep patching the SPs until all the SPs are gone? In this way, it doesn't matter whether it is executed before or after the `SPOD` is gone.

Sounds good. I agree that is our best option. I will go ahead and update the PR.